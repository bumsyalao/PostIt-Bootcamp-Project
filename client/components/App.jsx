 /*./client/components/App.js */

import React from 'react';

class App extends React.Component{
  render() {
    return (
      <div style ={{textAlign: 'center'}}>
        <h1>Welcome to Post-It</h1>
      </div>);
  }
}