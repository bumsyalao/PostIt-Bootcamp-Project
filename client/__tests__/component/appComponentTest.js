/* global expect jest test */
import React from 'react';
import { shallow, configure } from 'enzyme';
import toJson from 'enzyme-to-json';
import adapter from 'enzyme-adapter-react-16';
import App from '../../components/App';

configure({ adapter: new adapter() });

describe('App component', () => {
  const spy = jest.fn();
  beforeEach(() => {
    global.Materialize = { toast: jest.fn() };
  });
  afterEach(() => {
    spy.mockReset();
  });
  const component = shallow(
    <App />
  );

  test('should match the App component snapshot', () => {
    const tree = toJson(component);
    expect(tree).toMatchSnapshot();
  });
});
