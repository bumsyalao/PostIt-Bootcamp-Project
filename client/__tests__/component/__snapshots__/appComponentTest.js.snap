// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should match the App component snapshot 1`] = `
<div
  className="container"
>
  <Connect(withRouter(NavigationBar)) />
</div>
`;
